src/handwritten/_install_daqmx.py

```diff
@@ -325,8 +302,8 @@ def installdriver() -> None:
             logging.info("Windows platform detected")
             _install_daqmx_windows_driver()
         else:
-            print(f"installdriver command is supported only on Windows")
+            raise click.ClickException(f'Error installdriver command is supported only on Windows OS.')
     except Exception as e:
         logging.info("Error: %s", e)
         logging.debug(traceback.format_exc())
-        raise click.ClickException(f'Error occurred during installdriver') from e
+        raise click.ClickException(f'Error occurred during installdriver') from e
```

Comment:
Does this print the error details when logging is disabled or does it hide the details? If it hides the details, this error message is not going to make users successful. They are going to have to rerun the install with `-v`, and this error message doesn't tell them how to do that.

Reply:
@bkeryan 
Shall I rephrase the error message something like this
"An error occurred during the installation of the driver. Please rerun the install command with verbose(-v) logging enabled to debug.
@zhindes any suggestions how you want to improve this error message ?

Reply:
I think Brad's suggestion is that an error should be logged in a way that is hidden. Your default logging level is warning, so info/debug won't be shown. Line 307 should at least be a `logging.error`. I am guessing Python's normal exception handling with print the traceback for you (unless you catch this later and squelch it), so 308 is probably fine to leave as-is or remove. But you can try all this and make sure the behavior is good. Try to induce an error of an install in NIPM. Perhaps killing the NIPM process during one of the selection dialogs would do it?

Reply:
@bkeryan and @zhindes , I have tested an exception case by changing logging level and observed that current code is good.
We will document how to enable verbose logging along with the installation instructions [here](https://nidaqmx-python.readthedocs.io/en/latest/), so that users will know how to enable debugging when an error is encountered.

Reply:
I downloaded your branch so I can test the error handling. It's still not right.

If I change the metadata file to specify an invalid version like "asdf", here's what I get:
```
PS D:\dev\nidaqmx-python> poetry run nidaqmx installdriver
Error: An error occurred during the installation of the driver :Invalid version format found:invalid literal for int() with base 10: 'asdf'
```

- The spacing around the colons is wrong.
- The top-level try-except is catching ClickException and wrapping it with another ClickException.

Also, please be aware that string interpolation calls str() for you.